Index: ImHex-1.20.0/cmake/build_helpers.cmake
===================================================================
--- ImHex-1.20.0.orig/cmake/build_helpers.cmake
+++ ImHex-1.20.0/cmake/build_helpers.cmake
@@ -354,8 +354,7 @@ function(downloadImHexPatternsFiles dest
 
         FetchContent_Declare(
             imhex_patterns
-            GIT_REPOSITORY https://github.com/WerWolv/ImHex-Patterns.git
-            GIT_TAG master
+	    SOURCE_DIR @imhex_patterns_src@
         )
 
         FetchContent_Populate(imhex_patterns)
